src/_micro-api/libraries/signalDecoder/src/signalDecoder.cpp

```diff
@@ -993,10 +983,10 @@ const bool ManchesterpatternDecoder::doDecode() {
 						int pClock = abs(pdec->pattern[pdec->message[l-1]]);
 
 						// check if level is different
-						if (pClock < maxPulse && ((pdec->pattern[pdec->message[l-1]] > 0) != (pdec->pattern[pdec->message[l]] > 0))) {
+						if (pClock < maxPulse && sign(pdec->pattern[pdec->message[l-1]]) != sign(pdec->pattern[pdec->message[l]])) {
```

Comment:
schau mal in dodetect()
`		valid = (messageLen==0 || (*first ^ *last) < 0); // true if a and b have opposite signs
`

Wenn beim Puffer Kopieren nicht was verbaselt wird, dann kann es diese Situation nicht geben.
Hast Du festgestellt, dass manchmal gleiche Vorzeichen kommen?